src/devSVG.cpp

```diff
@@ -241,55 +290,59 @@ void svg_clip(double x0, double x1, double y0, double y1, pDevDesc dd) {
 }
 
 void svg_new_page(const pGEcontext gc, pDevDesc dd) {
+BEGIN_RCPP
```

Comment:
Oh, this is because `Rcpp::stop()` throws an exception, and it must be captured by Rcpp. `svg_new_page()` is a device function, which will be directly called by R and won't be protected by Rcpp without such macros.